ui/raidboss/data/05-shb/alliance/the_tower_at_paradigms_breach.js

```diff
@@ -0,0 +1,490 @@
+      durationSeconds: 4,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Attack Hansel',
+          de: 'Hänsel angreifen',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Hansel Upgraded Shield',
+      netRegex: NetRegexes.startsUsing({ id: '5C6B', source: 'Hansel', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5C6B', source: 'Hänsel', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5C6B', source: 'Hansel', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5C6B', source: 'ヘンゼル', capture: false }),
+      durationSeconds: 4,
+      infoText: (data, _, output) => output.text(),
+      outputStrings: {
+        text: {
+          en: 'Attack Gretel',
+          de: 'Gretel angreifen',
+        },
+      },
+    },
+    {
+      id: 'Paradigm Hansel/Gretel Wail',
+      netRegex: NetRegexes.startsUsing({ id: '5C7[67]', source: ['Hansel', 'Gretel'], capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5C7[67]', source: ['Hänsel', 'Gretel'], capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5C7[67]', source: ['Hansel', 'Gretel'], capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5C7[67]', source: ['ヘンゼル', 'グレーテル'], capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 4,
+      response: Responses.aoe(),
+    },
+    {
+      id: 'Paradigm Hansel/Gretel Crippling Blow',
+      netRegex: NetRegexes.startsUsing({ id: '5C7[89]', source: ['Hansel', 'Gretel'] }),
+      netRegexDe: NetRegexes.startsUsing({ id: '5C7[89]', source: ['Hänsel', 'Gretel'] }),
+      netRegexFr: NetRegexes.startsUsing({ id: '5C7[89]', source: ['Hansel', 'Gretel'] }),
+      netRegexJa: NetRegexes.startsUsing({ id: '5C7[89]', source: ['ヘンゼル', 'グレーテル'] }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.tankBuster(),
+    },
+    {
+      id: 'Paradigm Hansel/Gretel Seed Of Magic Alpha',
+      netRegex: NetRegexes.headMarker({ id: '0060' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.spread(),
+    },
+    {
+      id: 'Paradigm Hansel/Gretel Riot Of Magic',
+      netRegex: NetRegexes.headMarker({ id: '003E' }),
+      condition: Conditions.targetIsYou(),
+      durationSeconds: 4,
+      response: Responses.stackMarker(),
+    },
+    {
+      id: 'Paradigm Red Girl Cruelty',
+      netRegex: NetRegexes.startsUsing({ id: '6012', source: 'Red Girl', capture: false }),
+      netRegexDe: NetRegexes.startsUsing({ id: '6012', source: 'Rot(?:e|er|es|en) Mädchen', capture: false }),
+      netRegexFr: NetRegexes.startsUsing({ id: '6012', source: 'Fille En Rouge', capture: false }),
+      netRegexJa: NetRegexes.startsUsing({ id: '6012', source: '赤い少女', capture: false }),
+      condition: Conditions.caresAboutAOE(),
+      durationSeconds: 4,
```

Comment:
There's a lot of durationSeconds: 4 here.  Is there a reason to override the default of 3 in so many places?

Reply:
Doesn't have to, just felt more appropriate since the cast times were pretty long (in my opinion)